pymc/sampling_jax.py

```diff
@@ -6,6 +6,8 @@
 
 from typing import Callable, List, Optional
 
+from pymc.sampling import _init_jitter
```

Comment:
I think that this is ok for now, but in the long run I think that `sampling` should be a directory. It could house the code for the initial point selection (e.g. `_init_jitter`), and sampling specific to our python samplers, future aehmc, numpyro, blackjax or whatever comes along.